Components/Avatar/avatar.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/component.css">
+    <title>Avatar</title>
+</head>
+<body>
+    <h1>Avatar</h1>
+    <div>
+        <img src="/Assets/developer.png"  class="avatar avatar-sm" alt="" srcset="">
```

Comment:
You should provide the `alt` attribute in the image tag. Else looks great 👍

Reply:
Thankyou for pointing it out. I will correct it ASAP. Thankyou for your review.